README.md

```diff
@@ -1,19 +1,38 @@
-## Virtualenv usage
+# Foxglove Studio WebSocket protocol libraries
+
+This repository provides a protocol specification and reference implementations enabling [Foxglove Studio](https://github.com/foxglove/studio) to ingest arbitrary “live” streamed data.
+
+The protocol is encoding-agnostic, i.e. it can support Protobuf messages, ROS 1 or 2 messages, etc. (as long as the desired encoding is supported by both client and server).
```

Comment:
might be worth calling out that while the protocol is encoding-agnostic, Foxglove Studio only supports a specific set of encodings

Reply:
that's kinda what I was going for with the parenthetical but maybe it's not clear....I guess in the future this can point to a page on foxglove.dev/docs

Reply:
fair. yeah we can link to official message docs once they exist